cellmlmanip/mathml2sympy/transpiler.py

```diff
@@ -0,0 +1,440 @@
+"""
+Parses Content MathML and returns equivalent SymPy expressions
+
+Content Markup specification: https://www.w3.org/TR/MathML2/chapter4.html
+
+TODO: Validate MathML input
+"""
+from xml.dom import minidom, Node
+
+import sympy
+
+
+def parse_string(xml_string):
+    """
+    Reads MathML content from a string and returns equivalent SymPy expressions
+    """
+    dom = minidom.parseString(xml_string)
+    return parse_dom(dom.childNodes[0])
+
+
+def parse_dom(math_dom_element):
+    """
+    Accepts a <math> node of DOM structure and returns equivalent SymPy expressions.
+    Note: math_dom_element must point the <math> XmlNode, not the root XmlDocument
+
+    TODO: adding cellml namespace here, but really should be provided by caller
```

Comment:
As mentioned, we should enforce that this is given in parsed strings. (#6)

Reply:
Done